strings-csharp/GenerateStreamFromStringInCSharp/GenerateStreamFromStringInCSharp/GenerateStreamFromStringMethods.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Text;
+
+namespace GenerateStreamFromStringInCSharp
+{
+    public static class GenerateStreamFromStringMethods
+    {
+        public static Stream GenerateStreamFromString(string sampleString, Encoding? encoding = null)
+        {
+            encoding ??= Encoding.UTF8;
+
+            using var stream = new MemoryStream(encoding.GetByteCount(sampleString));
+            using var writer = new StreamWriter(stream, encoding);
+            writer.Write(sampleString);
+            stream.Position = 0;
+
+            return stream;
+        }
+
+        public static Stream ConciselyGenerateStreamFromString(string sampleString, Encoding? encoding = null)
+            => new MemoryStream((encoding ?? Encoding.UTF8).GetBytes(sampleString ?? string.Empty));
```

Comment:
in addition to @jshergal review, I'd like to suggest we write this as a full body method instead of an expression body method, and we should have a line there that clearly shows the conversion to bytes. 
Altough this is concise in term of number of lines but it's doing a lot of things in one line and it may confuse the reader.